src/main/java/ch/heigvd/dai/commands/Root.java

```diff
@@ -0,0 +1,66 @@
+package ch.heigvd.dai.commands;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    description = "",
```

Comment:
No description

Reply:
Nice catch! I'll fix this! Did you know you can also use the suggestion feature to propose a change directly?

```suggestion
    description = "shadow - a CLI tool to hide files inside BMP images",
```